### DISCUSSION TITLE
Is it possible to use a markdown list with Annotations?

### DISCUSSION THREAD

**Discussion by TRaSH-** [squidfunk/mkdocs-material#7200] [Q&A] [answered]
Currently i've the following 

![image](https://github.com/squidfunk/mkdocs-material/assets/6155095/6258b99d-d509-4a40-a109-a788892398c9)

I would like to know if it would be possible to set it up so it looks more like

![image](https://github.com/squidfunk/mkdocs-material/assets/6155095/80bd0661-ea22-4138-ae6c-7b5f4fb7e406)

**Comment by TRaSH-** [comment]
I got it working when I use the following

```md
1.  - ^^**Announced**^^: Radarr shall consider movies available as soon as they are added to Radarr. This setting is recommended if you have good private trackers that do not have fakes.
    - ^^**In Cinemas**^^: Radarr shall consider movies available as soon as movies hit cinemas (Theatrical Date on TMDb) This option is not recommended.
    - ^^**Released**^^: Radarr shall consider movies available as soon as the Blu-Ray or streaming version is released (Digital and Physical dates on TMDb) This option is recommended and likely should be combined with an Availability Delay of `-14` or `-21` days.
```

But the build fails because of the following rule `MD030/list-marker-space Spaces after list markers [Expected: 1; Actual: 2]` :(

**Reply by kamilkrzyskow** [reply]
Hello @TRaSH-,
considering that the annotations are a JavaScript client side feature, have you tried using raw HTML to define the list instead of usiong Markdown lists?

**Comment by TRaSH-** [comment, marked as answer]
No i actually haven't, but i did found another way that worked
https://github.com/TRaSH-Guides/Guides/pull/1947#issuecomment-2123528977

by using: https://github.com/DavidAnson/markdownlint/blob/main/README.md#configuration
